src/clarity-angular/dropdown/_dropdown.clarity.scss

```diff
@@ -85,18 +85,6 @@ $dropdown-white: clr-getColor(lightest);
         visibility: hidden;
         z-index: map-get($clr-layers, dropdown-menu);
 
-        .dropdown-menu {
-            // TODO: this is a hack to prevent breaking changes as we implement
-            // the popover directive. Remove this after the < 0.9.0 markup for
-            // dropdown (angular component) is deprecated.
```

Comment:
Hmm I think you misunderstand; the code changes in this PR made it possible to remove the hack and the deprecated feature still works as expected. But, I'll leave this comment as is, as we are still undergoing API design changes.